content/tutorials/git_zenodo/index.md

```diff
@@ -0,0 +1,195 @@
+Zenodo is aimed at preserving the deposits for the long term, which is great for reproducibility purposes.
+Also, it keeps multiple versions of your stuff together under the same record.
+Moreover, it provides a stable DOI link for each version and for each record as whole!
+
+## Integrating GitHub and Zenodo
+
+Just having a public code repository at GitHub is not enough for long-term reproducibility.
+This is where Zenodo comes in!
+From within Zenodo, you can setup a webhook in the GitHub repository which is triggered when you make a new release at GitHub.
+Then, GitHub will send the entire repository snapshot to Zenodo where it will be published as a zip-file.
+
+Also, Zenodo will increment the version number or take that (and other metadata) from a `.zenodo.json` metadata file, if you provide such a file in the repository.
+Do know that you can still manually edit all _metatadata_ of an existing publication at the Zenodo website (authors, title, description, version number, etc.).
+See further to discover this 'Edit' button!
+You cannot change the uploaded data themselves however, since a Zenodo publication is fixed (stable in time).
+Changing data is what _versions_ are meant for.
+
+Let's suppose you already have a GitHub repository and you want to set up such integration.\
+Let's do it!
+
+Oh, wait!\
+Did you know that you _cannot_ just _undo_ a publication at Zenodo?
+So make sure to get familiar first!
+
+If you are just taking your first steps and you want to do a safe exercise,
+then replace '<https://zenodo.org>' by '<https://sandbox.zenodo.org>'!
+Consider the Zenodo Sandbox a playground for your exercise.
+
+The Zenodo Sandbox creates the publications at <https://sandbox.zenodo.org>, e.g. <https://sandbox.zenodo.org/record/1139470>, but those are not perpetual and the displayed DOI's will not work (since they aren't actually created).
+
+First of all, make an account at <https://zenodo.org> (or the Sandbox!) and log in.
+
+#### Step 1: in Zenodo, go to 'GitHub' in your account settings
+
+![](images/settingsdropdown.png)\
+_Go to the GitHub settings in Zenodo._
+
+#### Step 2: find your repository, and flip the toggle to 'on'
+
+![](images/repositories.png)\
+_Your list of repositories._
+
+![](images/toggle.png)\
+_Flip the toggle!_
+
+Now you are ready for your first release.
+
+## Releasing at GitHub triggers a Zenodo publication
+
+#### Step 1: draft a new release
+
+In GitHub, go to the releases page, and click 'Draft new release'.
+Alternatively, extend the repository URL with `/releases/new`.
+
+![](images/draft.png)
+
+You get a page where you can:
+
+- select or create a tag.
+This is required: releases always refer to a tag.
+A Git tag is a label that is fixed to a specific snapshot (commit).
+It will typically contain a version number that you decide yourself.
```

Comment:
I recommend to elaborate a bit on good git tags.
```suggestion
It will typically contain a version number that you decide yourself (e.g. `v2` or `v2023`).
```